<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobItem</class>
 <widget class="QDialog" name="JobItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_Id">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_type">
        <property name="text">
         <string>Type of Item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_item">
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_Id"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Name"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_type"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_item"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
